low-level/framework/sspl_ll_d

```diff
@@ -794,6 +795,8 @@ if __name__ == "__main__":
     signal.signal(signal.SIGUSR1, set_log_level)
     # Handle signal for iem file refresh
     signal.signal(signal.SIGUSR2, refresh_iem_log_file)
+    # Handle signal to stop, save and generate coverage report
+    signal.signal(signal.SIGRTMIN, generate_cov_report)
```

Comment:
same comment as above

Reply:
Also, any reason why SIGRT* signal is being used? why not make reuse of SIGUSR1 as that will get freed up now, with log level dynamic change notification would come from conf store in future

Reply:
used SIGUSR1.